api-channel-issue-tracker/src/main/java/com/synopsys/integration/alert/api/channel/issue/search/enumeration/IssueCategory.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * api-channel-issue-tracker
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.api.channel.issue.search.enumeration;
+
+public enum IssueCategory {
```

Comment:
Is there a good example of us actually using this data after we set it? Or is it meant entirely as metadata and serialized into the created issue?

Reply:
The original ticket did not include what we would be using the issue category for. I found an example for using the IssueStatus in order to better differentiate between ADD/DELETE/UPDATE operations. I'm not sure what the intention for IssueCategory is in the future yet and was working under the assumption future tickets will be created to implement it.